### PR TITLE
Add chartkick v1.4.1 with git autoupdate, close #5739

### LINKED ISSUE #5739: [Request] Add Chartkick.js
**Library name: Chartkick** 
**Git repository url: https://github.com/ankane/chartkick.js**
**License(s): MIT**
**Official homepage: http://chartkick.com**

### PR DESCRIPTION
**Pull request by Piicksarn**
**`git repository : `** https://github.com/ankane/chartkick.js/tree/v1.4.1
**`Watch : 33  /  Star : 585 /  Fork : 76`**

- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct

### CODE REVIEW

**Comment by Piicksarn** [comment]
@Amomo, would you mind to have a look at the pr?
Thank you!~ :smiley:

**Comment by Amomo** [comment]
@Piicksarn I think you need to add `chartkick.js` also.

**Comment by Piicksarn** [comment]
@Amomo, sorry about that!!
would you mind to check it again?
Thank you!

**Comment by Amomo** [comment]
@Piicksarn please rebase it to match the latest master branch~

**Comment by Piicksarn** [comment]
@Amomo, i have the pr rebased to master!
would you mind to have a look at the pr again?~
Thank you ~~~

**Comment by Amomo** [comment]
@Piicksarn sorry I didn't notice that you had tagged me... 😭
Could you rebase it again? Thank you!

**Comment by Amomo** [comment]
Oh! And, could you `cc` the author in the commit message?
It would be good to notify him this lib is adding to cdnjs!

**Comment by Piicksarn** [comment]
@Amomo, I have it rebased and need your help for checking!
I will cc the author after the pr been merged! :smiley:

**Comment by Amomo** [comment]
@Piicksarn I mean you can cc the author in your commit message.
Once your PR is merged, it would notify the author right away~

@PeterDaveHello I think this PR is fine, could you take a look? Thanks! :-D

**Comment by PeterDaveHello** [comment]
where is your cc part?

**Comment by Piicksarn** [comment]
@Amomo, would you mind to checkout the pr again?
Thank you!

**Comment by Amomo** [comment]
@PeterDaveHello I think this PR is fine, could you take a look? Thank you!

**Comment by PeterDaveHello** [comment]
Wrong auto-update config ...

**Comment by Piicksarn** [comment]
@Amomo, would you please have a look at the pr?
Thank you!

**Comment by Piicksarn** [comment]
cc @Amomo

File: ajax/libs/chartkick/package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "chartkick",
+  "filename": "chartkick.min.js",
+  "version": "1.4.1",
+  "homepage": "https://github.com/ankane/chartkick.js",
+  "description": "Create beautiful Javascript charts with minimal code",
+  "keywords": [
+    "charts"
+  ],
+  "authors": [
```
Comment (Amomo) [inline ajax/libs/chartkick/package.json]: @Piicksarn 
There is only one author, I think using "author" field with a string would be nicer.

And, could you give this library more keywords? Thanks. :-)

**Comment by PeterDaveHello** [comment]
Added in #7413

**Pull request closed without merging**